.github/workflows/release.yml

```
@@ -0,0 +1,98 @@
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+
+      - name: Run tests
+        run: pnpm test
+
+      - name: Build all packages
+        run: pnpm build
+
+      - name: Determine version
+        id: version
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:6:28: Double quote to prevent globbing and word splitting

(shellcheck)